waiting-for-finish.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+echo "waiting for all background process to stop..."
```

Comment:
See http://unix.stackexchange.com/questions/65803/why-is-printf-better-than-echo.